### ISSUE TITLE
New Icon Request - Floor/Room Layout

### ISSUE THREAD

**Issue by leroyuk1983** [FortAwesome/Font-Awesome#12064]
I would like an icon to be used with my floor/room layout plans.

Something similar to this
![unique-floor-plans](https://user-images.githubusercontent.com/35264077/34724548-185bc38a-f546-11e7-96ba-334b7188e5bd.png)

**Comment by BasicSoft** [comment]
+1
![17576-200](https://user-images.githubusercontent.com/25501553/38352995-62402498-38b5-11e8-859b-75674942c1f9.png)
![download](https://user-images.githubusercontent.com/25501553/38353046-8ab101a4-38b5-11e8-9677-9ba3f0002069.png)

**Comment by Portekoi** [comment]
Room icon like this will be great : 
https://thenounproject.com/term/room/20015/

**Comment by Portekoi** [comment]
Like this one? https://thenounproject.com/term/room/20015/

**Comment by BasicSoft** [comment]
Better this one: 
https://thenounproject.com/term/floor-plan/29508/
or
https://thenounproject.com/term/room/25764/

For more detailed icons one of these would be ok:
https://thenounproject.com/term/room/125128/
https://thenounproject.com/term/room/125129/

**Comment by lucanos** [comment]
I'd like to see metric and imperial versions like this
![screen shot 2018-10-25 at 1 34 33 pm](https://user-images.githubusercontent.com/126774/47473002-7ff2a780-d85c-11e8-8fb6-74cd0b8a20a6.png)

**Comment by rept** [comment]
> I'd like to see metric and imperial versions like this

Wouldn't that be another icon, for example: surface

**Comment by SherinBloemendaal** [comment]
Would be awesome! Status?

**Comment by lucanos** [comment]
> > I'd like to see metric and imperial versions like this
> 
> Wouldn't that be another icon, for example: surface

No. If we have a floorplan icon, then having a variant for floor space would be nice, with imperial and metric versions.

A generic "surface" icon might work in some cases, but for things like real estate listings, I could see the floor space icon being used to indicate interior space, and the "surface" icon being for the lot size, as an example.

**Comment by LeonSpors** [comment]
Any updates on this?

**Comment by michaelpaps** [comment]
This would be useful for real estate projects. 

Usage example: [icon] Download Floor plan

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
